components/Buttons/examples/ButtonsContentEdgeInsetsExample.m

```diff
@@ -41,8 +41,9 @@ + (NSDictionary *)catalogMetadata {
 
 #pragma mark - UIViewController
 
-- (id)init {
-  self = [super init];
+- (instancetype)initWithCoder:(NSCoder *)coder
```

Comment:
Our team norm is to implement all designated initializers and call a `common<#ClassName#>Initializer` method; this will reduce the likelihood of similar errors in the future. I believe we should be able to get by with just implementing initWithNibName:... and initWithCoder: